lib/membrane/time.ex

```diff
@@ -179,6 +182,41 @@ defmodule Membrane.Time do
     DateTime.from_unix!(value |> nanoseconds, :nanosecond)
   end
 
+  @doc """
+  Converts NTP timestamp (time since 0h on 1st Jan 1900) into Unix timestamp
+  (time since 1st Jan 1970) represented in `Membrane.Time` units.
+
+  NTP timestamp uses fixed point representation with the integer part in the first 32 bits
+  and the fractional part in the last 32 bits.
+  """
+  @spec from_ntp_timestamp(ntp_time :: <<_::64>>) :: t()
+  def from_ntp_timestamp(<<ntp_seconds::32, ntp_fraction::32>>) do
+    fractional =
+      ntp_fraction
+      |> Ratio.new(Ratio.pow(2, 32))
+      |> Ratio.mult(second())
+      |> Ratio.trunc()
```

Comment:
I have figured out another way to do this:
More precise and possibly faster,  but hacky. Do you think it's too hacky to include @mat-hek?
```suggestion
    <<fraction_ieee::float-size(64)>> = <<0::1, 1023::11, ntp_fraction::32, 0::20>>
    fractional = trunc((fraction_ieee - 1) * second())

```